Docs/sequelize-cli_shortcut.md

```diff
@@ -17,3 +17,7 @@
 - Revert back to initial state by undoing all migrations
 
         npx sequelize-cli db:migrate:undo:all
+
+- Append table with migration [see also doc](https://sequelize.readthedocs.io/en/latest/docs/migrations/)
```

Comment:
nit: "append" is a bit odd to me.   maybe something like `Change the database with migration`?